docs/docs/scanner/vulnerability/language/nodejs.md

```diff
@@ -29,6 +29,8 @@ Trivy parses `package-lock.json`.
 To identify licenses, you need to download dependencies to `node_modules` beforehand.
 Trivy analyzes `node_modules` for licenses.
 
+By default, Trivy doesn't report developer dependencies. Use the `--include-dev-deps` flag to include them.
```

Comment:
https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file

```suggestion
By default, Trivy doesn't report development dependencies. Use the `--include-dev-deps` flag to include them.
```

Reply:
Changed in https://github.com/aquasecurity/trivy/pull/4700/commits/f4a1e2002f31d401323e09fb747d58c57b97d4a3